lib/application.js

```diff
@@ -284,7 +285,10 @@ app.handler = function(type, options) {
   var handler = this._handlers[type] = remotes.handler(type, options);
 
   remotes.classes().forEach(function(sharedClass) {
-    sharedClass.ctor.emit('mounted', app, sharedClass, remotes);
+    /* HACK!!! */
+    if (sharedClass.name !== 'ctrl') {
```

Comment:
I don't think we use the "mounted" event anywhere else besides explorer as described below, but I am not sure. Can you do a quick `grep` to confirm?